wasm2c/examples/build-standlone/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+if [ -z "$1" ]
+  then
+    echo "Compiles wasm file into standalone binary"
+    echo "usage: build.sh foo.wasm"
+    echo "outputs: foo.elf"
+    exit
+fi
+
+ORIGIN_DIR=$(pwd)
+SCRIPT_DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+output_file=$(basename -- "$1")
+output_file="${output_file%%.*}.elf"
+
+set -v
+
+cd $SCRIPT_DIR
+make clean
+
+cp ${ORIGIN_DIR}/$1 ./input.wasm
+make input.c
+
+make input.elf
```

Comment:
I really like this `build-standalone` idea.   The mixing of a build.sh script and the Makefile file a find a little confusing, but I think we can iterate on it over time.

Reply:
For sure, it was just a quick way to get something working, if we want to promote to a first class feature I can just replace it with a stand-alone script.